health_check/backends.py

```diff
@@ -11,6 +11,9 @@
 class BaseHealthCheckBackend:
     def __init__(self):
         self.errors = []
+        self.warnings = []
+        self.critical = getattr(self, 'critical', True)
```

Comment:
https://github.com/KristianOellegaard/django-health-check/pull/191/files#diff-88b99bb28683bd5b7e3a204826ead112R177

Simply, it determines if that attribute has been set on the object, since that value would be preferred if so, otherwise it sets it to the default.